client/src/partials/patients/groups/groups.js

```diff
@@ -1,127 +1,126 @@
+/** declaring the controller to the bhima module **/
 angular.module('bhima.controllers')
-.controller('patientGroup', [
-  '$scope',
-  '$translate',
-  '$window',
-  '$location',
-  'connect',
-  'validate',
-  'appstate',
-  'messenger',
-  'uuid',
-  function ($scope, $translate, $window, $location, connect, validate, appstate, messenger, uuid) {
-    var dependencies = {};
-    var session = $scope.session = {
-      selected : null
-    };
-
-    dependencies.group = {
-
-       query : {
-        identifier : 'uuid',
-        tables : {
-          'patient_group' : {columns : ['uuid', 'name', 'price_list_uuid', 'subsidy_uuid']}
-        }
-      }
-    };
-
-    dependencies.subsidies = {
-      query : {
-        identifier : 'uuid',
-        tables : {
-          'subsidy' : {columns : ['uuid', 'text']}
-        }
-      }
-    };
-
-    dependencies.list = {
-      query : {
-        identifier : 'uuid',
-        tables : {
-          'price_list' : {
-            columns : ['uuid', 'title']
-          }
-        }
-      }
-    };
-
-    appstate.register('enterprise', loadEnterprise);
-
-    function loadEnterprise(enterprise) {
-      $scope.enterprise = enterprise;
-      dependencies.group.query.where = ['patient_group.enterprise_id='+enterprise.id];
-      validate.process(dependencies).then(initialisePatientGroup);
-    }
+.controller('patientGroupController', patientGroupController);
 
-    function initialisePatientGroup(model) {
-      angular.extend($scope, model);
-    }
+/** inject service(s) or dependencies for the modules **/
+patientGroupController.$inject = ['PatientGroupService', 'PriceListService', 'SessionService', '$window'];
 
-    $scope.remove = function (grp) {
-      if (!$window.confirm($translate.instant('PATIENT_GRP.CONFIRM_MESSAGE'))) {
-        return;
-      }
+function patientGroupController (patientGroupService, priceListService, sessionService, $window){
```

Comment:
As I understand controllers are formatted as UpperCaseController. 

Whatever the case this should be uniform across BHIMA.